code/include/z3D/z3D.h

```diff
@@ -700,22 +694,22 @@ typedef struct MainClass {
 extern GlobalContext* gGlobalContext;
 extern const u32 ItemSlots[];
 extern const char DungeonNames[][25];
-#define gSaveContext (*(SaveContext*)0x00587958)
+#define gSaveContext (*(SaveContext*)GAME_ADDR(0x587958))
 #define gStaticContext (*(StaticContext*)0x08080010)
```

Comment:
You missed one address here :)
After that, lgtm!

Reply:
I intentionally excluded that since it's outside the range that can be seen in Ghidra, so I'm just assuming that plays by different rules than these normal addresses. Same with the other 8-symbol long addresses. Though, those are all above `0x10000000`. Do you think `gStaticContext` should be included anyway?

Reply:
Oh, you're right, I didn't think about that. Let's leave it like this then.